manifests/fastd.pp

```diff
@@ -26,6 +28,15 @@
       ensure => file,
              notify => Service[ffnord::resources::fastd],
              content => template('ffnord/etc/fastd/fastd.conf.erb');
+    "/etc/fastd/${mesh_code}-mesh-vpn/fastd-blacklist.sh":
+      ensure => file,
+             notify => Service[ffnord::resources::fastd],
+             content => template('ffnord/etc/fastd/fastd-blacklist.sh.erb'),
+             mode => '0766';
+   "/etc/fastd/${mesh_code}-mesh-vpn/fastd-blacklist.json":
```

Comment:
I find this a bit difficult; fastdbl is a different project and THEY suggest already how to deal with things (which, frankly, we don't do as of now out of lazyness -- which is bad, I know, and will bite us twice if we finally need to synchronize across gateways, I know). My stub code is just to enable it in this puppetization effort. In principle, I do agree that it should be setup in full from puppet (which would mean *I* would need to add at least the stub file in git, so we actually would have that automated already at the time we'd need it ;)). On a more generic level I object the (mis-) use of github/some git-server as poor man's rsync replacement. Is there ever a need to go back to last month' version of blacklist/fastd peers? I can't imagine one. Only because it's convenient, git(hub) shouldn't be abused like that. IMHO, YMMV ;) (And no, I don't have a more handy, more easy to use solution to suggest ...)

That said, I most likely add synchronisation (via git) of the blacklist as well some afternoon, just because it actually makes sense. *sigh*